Element/Assets/ThirdParty/Singleton.cs

```diff
@@ -1,64 +1,28 @@
 ﻿using UnityEngine;
 
-/// <summary>
```

Comment:
Originally this was a third party script, but I wanted to pass it off as something we wrote because "Singleton" is a design pattern.